web/toDoList.jsp

```diff
@@ -1,160 +1,147 @@
-                var idOfRemainder = '#' + this.id;
-                reminderFunction(idOfRemainder);
-            });
-            var listEmpty = <%=isListEmpty%>;
-            var idOfLastItem = 0;
-            $(".checkboxStyle").css('visibility', 'hidden');
-            var itemInputNode = $('#todoItemName');
-            itemInputNode.keypress(function (e) {
-                if (e.which == 13) {
-                    var itemName = itemInputNode.val();
-                    if (!itemName.trim()) {
-                        return;
-                    }
-                    $('#emptyListMessage').css("display", "none");
-                    $.ajax({
-                        url: 'addItem.jsp',
-                        data: {
-                            item: itemName,
-                        },
-                        method: 'POST'
-                    }).done(function (data) {
-                        idOfLastItem = parseInt(data);
-                        if (listEmpty) {
-                            listEmpty = false;
-                            $('#listBody').prepend("<div class='toDoListDataStyle' id='listItemId-" + idOfLastItem + "'><span contenteditable='true' id='item-" + idOfLastItem + "' class ='itemStyle'>" + itemName + " </span><span class='checkboxStyle' id='checkboxId-" + idOfLastItem + "' ><input type='checkbox'/></span><span id='remainderButton-" + idOfLastItem + "' class='button btn btn-default btn-sm remainderButtonStyle'>Reminder</span></div>");
-                            $('#checkboxId-' + idOfLastItem).css('visibility', 'hidden');
-                        } else {
-                            $('#listBody').last().append("<div class='toDoListDataStyle' id='listItemId-" + idOfLastItem + "'> <span contenteditable='true' id='item-" + idOfLastItem + "' class ='itemStyle'>" + itemName + " </span><span class='checkboxStyle' id='checkboxId-" + idOfLastItem + "' ><input type='checkbox'/></span><span id='remainderButton-" + idOfLastItem + "' class='button btn btn-default btn-sm remainderButtonStyle'>Reminder</span></div>");
-                            $('#checkboxId-' + idOfLastItem).css('visibility', 'hidden');
-                        }
-                        setUpClickListeners("#listItemId-" + idOfLastItem);
-                        updateItemFunction("#item-" + idOfLastItem);
-                        reminderFunction("#remainderButton-" + idOfLastItem);
-                    }).fail(function () {
-                    });
-                    itemInputNode.val("");
-                }
-            });
-            var setUpClickListeners = function (Id) {
-                mouseEventFunction(Id);
-                deleteFunction(Id);
-                if (($('#listBody').children().length) == 0) {
-                    listEmpty = true;
-                    $('#emptyListMessage').css("display", "block");
+</div>
+<script>
+    $(function () {
+        $('.datetimepicker').datetimepicker({});
```

Comment:
call this function at the end of the function please. it is okay to initialize `datetimepicker` with a small delay.